CountryPicker/CountryPicker/Country.swift

```diff
@@ -12,13 +12,13 @@ import UIKit
 open class Country {
 
     // MARK:- Variable
-    open var countryCode: String
+    open private(set) var countryCode: String
 
     /// Name of the country
-    open var countryName: String
+    open private(set) var countryName: String
     
     /// Dialing code of the country
-    open var dialingCode: String
+    open private(set) var dialingCode: String?
```

Comment:
reason ?

Reply:
Not all dialing codes exist in .plist file. I guess that can be changed.